internal/attestation/snp/validator.go

```diff
@@ -69,7 +68,7 @@ func NewValidatorWithCallbacks(optsGen validateOptsGenerator, kdsGetter trust.HT
 		validateOptsGen: optsGen,
 		callbackers:     callbacks,
 		kdsGetter:       kdsGetter,
-		logger:          slog.New(logger.NewHandler(log.Handler(), "snp-validator")),
```

Comment:
why are we removing the log subsystem?

Reply:
The log subsystem is already added on the caller side, so this would create a nested subsystem with the same name.